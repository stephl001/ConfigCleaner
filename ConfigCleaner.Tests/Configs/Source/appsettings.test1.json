{
  "Logging": {
    "LogLevel": "Debug",
    "LogFilePath": "logs/app.log"
  },
  "Database": {
    "ConnectionString": "Server=myserver;Database=mydb;User=devuser;Password=devpassword",
    "MaxConnections": 100,
    "TimeoutSeconds": 30.5,
    "IsEnabled": true,
    "BackupPath": null
  },
  "API": {
    "BaseUrl": "https://api.example.com",
    "ApiKey": "dev-api-key-here",
    "RetryCount": 5,
    "IsProduction": false
  },
  "Features": {
    "FeatureA:Enabled": false,
    "FeatureB:Threshold": 2.75,
    "FeatureC:Config": {
      "Setting1": "value1",
      "Setting2": 42,
      "Setting3": null
    }
  }
}