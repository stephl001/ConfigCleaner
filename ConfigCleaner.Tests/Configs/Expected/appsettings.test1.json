{
  "Logging": {
    "LogLevel": "Debug"
  },
  "Database": {
    "ConnectionString": "Server=myserver;Database=mydb;User=devuser;Password=devpassword"
  },
  "API": {
    "ApiKey": "dev-api-key-here",
    "RetryCount": 5,
    "IsProduction": false
  },
  "Features": {
    "FeatureA:Enabled": false,
    "FeatureB:Threshold": 2.75
  }
}